torchao/quantization/quant_api.py

```diff
@@ -487,9 +487,30 @@ def _int8_asymm_per_token_quant(x: torch.Tensor) -> torch.Tensor:
     else:
         return to_affine_quantized_intx(x, mapping_type, _get_per_token_block_size(x), target_dtype)
 
-def apply_int8_dynamic_activation_int4_weight_quant(weight, group_size=32, mapping_type=MappingType.SYMMETRIC):
+def _int8_symm_per_token_quant(x: torch.Tensor) -> torch.Tensor:
+    mapping_type = MappingType.SYMMETRIC
+    target_dtype = torch.int8
+    eps = 1e-5
+    quant_min = -127
+    quant_max = 127
+
+    return to_affine_quantized_intx(
+        x,
+        mapping_type,
+        _get_per_token_block_size(x),
+        target_dtype,
+        eps=eps,
+        quant_min=quant_min,
+        quant_max=quant_max,
+        scale_dtype=torch.float32,
+    )
+
+
+def apply_int8_dynamic_activation_int4_weight_quant(weight, group_size=32, layout=PlainLayout(), mapping_type=MappingType.SYMMETRIC, input_mapping_type=MappingType.ASYMMETRIC):
```

Comment:
nit: probably clearer if we also rename the first `mapping_type` to `weight_mapping_type`, but this can be a follow up since we need to be careful about BC

Reply:
Yes, I agree that this should be addressed later.